### PR TITLE
Remove mock from install_requires

### PR DESCRIPTION
**Pull request by breerly**
`mock` is only being used in tests.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,7 +51,6 @@ def run_tests(self):
     },
     install_requires=[
         'six>=1.7.3',
-        'mock==1.0.1',
```
Comment (blampe) [inline setup.py]:
```
test_requires=[
    'mock'
]
```

Comment (breerly) [inline setup.py]: They already have it specified elsewhere, and tests are passing.

Comment (kracekumar) [inline setup.py]: https://github.com/schematics/schematics/blob/development/tox.ini, https://github.com/schematics/schematics/blob/development/requirements-testing.txt

Comment (blampe) [inline setup.py]: :+1:

**Comment by kstrauser** [comment]
Works for me.

We need to lunch soon.

**Pull request merged**